File: api/src/main/java/sn/jmad/sonac/model/ClauseActe.java
package sn.jmad.sonac.model;

import java.io.Serializable;
import java.util.Date;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Table;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Entity
@Table(name = "clauseActe")
public class ClauseActe implements Serializable{
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	@Id
	private Long clact_id;
	private Long clact_numeroclause;
	private Long clact_numeroacte;
	private String clact_texte1;
	private String clact_texte2;

}

File: frontend/src/app/model/ClauseActe.ts
export class ClauseActe{
    clact_id: Number;
    clact_numeroclause: Number;
    clact_numeroacte: Number;
    clact_texte1: String;
    clact_texte2: String;
    constructor(
    clact_numeroclause: Number,
    clact_numeroacte: Number,
    clact_texte1: String,
    clact_texte2: String
    ) {
        this.clact_numeroclause = clact_numeroclause;
        this.clact_numeroacte = clact_numeroacte;
        this.clact_texte1 = clact_texte1;
        this.clact_texte2 = clact_texte2;
     }
}